### PR TITLE
Make a VDS (how to / how do we even) for VS-1026

### PR DESCRIPTION
**Pull request by RoriCremer**
Notebook documenting the _actual_ creation of this VDS here:
https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/analysis/launch/Make%20a%20VDS%20with%20Quickstart.ipynb?mode=edit

This VDS (well I made two, one with VQSR and one with VETS) can then be used for a tie out in the near future with the WDL pipeline for VDS creation

gs://fc-eada2674-7c2b-42a6-8db3-0246872596dc/quickstart-vds-for-wdl-tieout/VQSR-Classic/vds/

gs://fc-eada2674-7c2b-42a6-8db3-0246872596dc/quickstart-vds-for-wdl-tieout/VETS/vds/

### CODE REVIEW

**Review by rsasch — CHANGES_REQUESTED** [review]
I tried to run the notebook but it failed at the `download ./hail_gvs_import.py` step because it could not find the file at the location specified:

> curl --silent --location --remote-name https://raw.githubusercontent.com/broadinstitute/gatk/rc-vs-1025-hail-version-120/scripts/variantstore/wdl/extract/hail_gvs_import.py --output hail_gvs_import.py

**Comment by RoriCremer** [comment]
omg Bec you are right--I merged the branch that this file lived on!

**Comment by RoriCremer** [comment]
also worth noting that the current version of VDS creation does not seem to drop GQ0s on it's own

**Review by rsasch — APPROVED** [review]
After the changes to the notebook and our chat 👍🏻

File: scripts/variantstore/docs/aou/vds/Creating a VDS.md [hunk 1]
```diff
@@ -3,12 +3,28 @@
...
 ## Configuring the Terra notebook cluster
 
 Please follow the instructions
 in [AoU Delta VDS Cluster Configuration](cluster/AoU%20Delta%20VDS%20Cluster%20Configuration.md) to set up a Delta-scale
 cluster.
 
+## Using the created notebook, run the python script to create a VDS from the avro files created by the GvsExtractAvroFilesForHail WDL
+These avro files will be passed in using the --avro-path parameter
+The vds-path parameter tells the WDL where to write the final desired VDS to
+There is quite a bit of temporary data created and while it will be cleaned up later, it's helpful to hand the python script a location.
```
Comment (gbggrant) [inline scripts/variantstore/docs/aou/vds/Creating a VDS.md:18]: is it 'helpful' or is it required? I assume you are talking about the `temp-path` directory? (so clarify this line).

File: scripts/variantstore/docs/aou/vds/Creating a VDS.md [hunk 2]
```diff
@@ -3,12 +3,28 @@
...
+
 ## Configuring the Terra notebook cluster
 
 Please follow the instructions
 in [AoU Delta VDS Cluster Configuration](cluster/AoU%20Delta%20VDS%20Cluster%20Configuration.md) to set up a Delta-scale
 cluster.
 
+## Using the created notebook, run the python script to create a VDS from the avro files created by the GvsExtractAvroFilesForHail WDL
+These avro files will be passed in using the --avro-path parameter
+The vds-path parameter tells the WDL where to write the final desired VDS to
```
Comment (gbggrant) [inline scripts/variantstore/docs/aou/vds/Creating a VDS.md:17]:
```suggestion
The vds-path parameter tells the WDL where to write the final VDS to
```

**Review by gbggrant — APPROVED** [review]
LGTM. - a couple of minor comments on wording.

File: scripts/variantstore/docs/aou/vds/Creating a VDS.md (hunk 1 above)
Comment (RoriCremer) [reply scripts/variantstore/docs/aou/vds/Creating a VDS.md:18]: I think it's only needed if there is a lot of data, but better to require it when it's not needed then the opposite

**Pull request merged**